package.json

```diff
@@ -268,7 +268,7 @@
     "string-width": "^4.2.0",
     "strip-ansi-stream": "^1.0.0",
     "tar-stream": "^2.1.3",
```

Comment:
Maybe we should pin `tar-stream` here as well, however it was not needed during my testing. 🤔